### PR TITLE
Windows support

### PR DESCRIPTION
**Pull request by diorcety**

### CODE REVIEW

File: circus/process.py [hunk 1]
```diff
@@ -11,7 +11,8 @@
 import sys
 import errno
 import os
-from subprocess import PIPE
+from subprocess import PIPE, CREATE_NEW_PROCESS_GROUP
```
Comment (k4nar) [inline circus/process.py]: `CREATE_NEW_PROCESS_GROUP` isn't defined on non-linux hosts, so this should be in a try/except.

**Comment by k4nar** [comment]
Thanks! There seem to be an import error and some markup errors (you can run `flake8` locally to see them), but otherwise it looks great.

**Comment by diorcety** [comment]
Should be fixed now

**Comment by k4nar** [comment]
Those changes breaks the tests on Unix. Could you try to fix this?

**Comment by diorcety** [comment]
I will try soon.

**Comment by diorcety** [comment]
Travis seems a bit broken too... no?

**Pull request closed without merging**